javascripts/controllers/NewCtrl.js

```diff
@@ -1,5 +1,16 @@
 "use strict";
 
-app.controller("NewCtrl", function ($scope) {
-    $scope.controller = "NewCtrl";
+app.controller("NewCtrl", function ($location, $rootScope, $scope, ContactService) {
+
+    $scope.submitForm = () => {
+        let newContact = $scope.contact;
+        $scope.contact.uid = $rootScope.uid;
```

Comment:
shouldn't this be newContact.uid = $rootScope.uid?

Reply:
GTG